packages/notebook-extension/src/index.ts

```diff
@@ -3554,6 +3554,10 @@ function addCommands(
         (settings?.composite.windowingMode === 'full' ?? false);
       return enabled;
     },
+    isToggled: () => {
+      const current = tracker.currentWidget;
```

Comment:
I think this was the case at some point while iterating on this PR, but it was creating some issues because of the `activate` logic in that function, which was preventing switching to another widget in the dock panel:

https://github.com/jupyterlab/jupyterlab/blob/56a01218c7e67fc369cf44e34238493c121fc885/packages/notebook-extension/src/index.ts#L2145-L2149

Reply:
Ok, thanks for the clarification!